src/packages/cli/src/cli.ts

```diff
@@ -69,6 +76,7 @@ const handleSignal = async (signal: NodeJS.Signals) => {
 };
 const closeHandler = async () => {
   try {
+    if (versionCheck) versionCheck.destroy();
```

Comment:
Currently `versionCheck` will never be falsey, so this is fine:

```suggestion
    versionCheck.destroy();
```

Though if you end up doing `versionCheck.init().log().then(() => { versionCheck = null });` you'd want to keep the check in.